tasks/javascript.js

```diff
@@ -150,8 +150,7 @@ function setUpTaskJavascript(tyRun) {
       // give a single optimised javascript file in the build directory with source map for each
       gulp.task('javascript:build', function () {
         return streams.jsApp({read: false})
-          .pipe(browserify
-            .compile(80, getTransforms(!cliArgs.unminified))
+          .pipe(browserify(80, getTransforms(!cliArgs.unminified))
             .each(!cliArgs.unminified))
```

Comment:
For these lines, I think it is good to go for now, but for future-proofing reasons, I think passing in an options hash would be better, e.g.:

```javascript
  .pipe(browserify(80, getTransforms({ minified: !cliArgs.unminified }))
    .each({ minified: !cliArgs.unminified }))
```